client/src/Components/Robot.js

```diff
@@ -15,8 +15,18 @@ class Robot extends Component {
   }
 
   getImage() {
-    // TODO
-    return "Robot_2_-_Blue.png";
+    // Based on Scriabin's colour mapping
+    const colour_map = {
```

Comment:
Can we move the const out of the function so we don't recreate it every time the function is invoked?